plugins/synapticsmst/synapticsmst-device.h

```diff
@@ -82,6 +82,7 @@ gboolean	 synapticsmst_device_write_firmware		(SynapticsMSTDevice *device,
 								 GFileProgressCallback	progress_cb,
 								 gpointer user_data,
 								 gboolean reboot,
+								 gboolean install_force,
```

Comment:
normally two booleans is my prompt to move things to a bitfield -- do you see anything else being added in the future?

Reply:
I don't think there is anything else down the pipe, this wraps up the last TODO in synaptics MST plugin.  Unless you feel strongly how about if we end up getting a 3rd we do a bitfield otherwise keep it as is?

Reply:
Ohh, two is okay for me, but I'll push-back for three :)